src/select.rs

```diff
@@ -87,3 +92,40 @@ impl<'a> TElement<'a> for &'a ElementData {
         }
     }
 }
+
+impl<'a> Node<'a> {
+    pub fn css(&'a self, css_str: &str) -> FilterNodes<'a, Descendants<'a>> {
```

Comment:
Please rename this to `select`. Although they’re related, Selectors are not CSS.